nautobot/core/templatetags/helpers.py

```diff
@@ -24,6 +24,7 @@
 from nautobot.core.constants import PAGINATE_COUNT_DEFAULT
 from nautobot.core.utils import color, config, data, logging as nautobot_logging, lookup
 from nautobot.core.utils.requests import add_nautobot_version_query_param_to_url
+from nautobot.extras.choices import ApprovalWorkflowStateChoices
```

Comment:
Might make sense to add a new file under `nautobot/extras/templatetags/` rather than augmenting core `helpers` just so that we don't have to add yet another case of core depending on extras?